src/hooks/api/useGetDummyApi.ts

```diff
@@ -0,0 +1,39 @@
+import axios from 'axios';
+import { useMutation } from 'react-query';
+
+export interface Cat {
+  status: Status;
+  type: string;
+  deleted: boolean;
+  _id: string;
+  user: string;
+  text: string;
+  __v: number;
+  source: string;
+  updatedAt: string;
+  createdAt: string;
+  used: boolean;
+}
+
+interface Status {
+  verified: boolean;
+  sentCount: number;
+  feedback: string;
+}
+
+export const callApi = async () => {
```

Comment:
```suggestion
export const fetchCats = async () => {
```
요건 더미라서 그러신 걸 수도 있지만, callApi와 같이 의미를 알기 힘든 추상적인 이름보다는 `fetchCats`와 같이 명확한 이름이 더 좋을 것 같아요!

Reply:
변경했습니다 :) https://github.com/Lubycon/DEVTI-front/pull/18/commits/8e39ce85a9e53a8440bda420ddbf685bb1d4ea4c